shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -835,6 +877,9 @@ - (void)viewDidLayoutSubviews {
   CGSize viewSize = self.view.bounds.size;
   CGFloat scale = [UIScreen mainScreen].scale;
 
+  // Purposefully place this offscreen.
+  _scrollView.get().bounds = CGRectMake(0.0, -10.0, viewSize.width, 0);
```

Comment:
There is something magical about using this bounds as a way to still get events but to have it not be visible.  I've tried just using the frame to place it off screen and it doesn't work.  I've tried setting the y value to 0, but that causes it to stop working too.

Reply:
No longer necessary, started setting the frame which makes more sense.